python/foglamp/plugins/north/http_north/http_north.py

```diff
@@ -75,8 +75,13 @@ def plugin_init(data):
     return config
 
 
-def plugin_send(data, payload, stream_id):
-    return http_north.send_payloads(payload, stream_id)
+async def plugin_send(data, payload, stream_id):
+
```

Comment:
Do we need `data`, as it seems to be unused? And remove space line.

Reply:
it is needed, 
the sign of the plugin_send call should be the same for every plugin.